process.sh

```diff
@@ -33,12 +33,14 @@ main() {
     echo 'AWS:'
     if [[ "$CI" = 'true' ]] || [[ "$NO_PUSH" = 'true' ]]
     then
-        echo 'CI: Skip push to AWS'
+        echo 'CI: Skip push to AWS and GCS'
     else
         # Exclude the *HUGE* PNGs in the base directory:
         # The tiles for S3 are in subdirectories;
         # We keep the PNGs around because it takes a long time to generate them.
-        aws s3 cp --exclude "$OUTPUT/*.ome.tif" --recursive "$OUTPUT" s3://"$S3_TARGET"
+        TILES_BASE='vanderbilt_MxIF.images'
+        aws s3 cp --exclude "$OUTPUT/*.ome.tif*" --recursive "$OUTPUT" s3://"$S3_TARGET"
```

Comment:
Should the first argument below (`$OUTPUT/$TILES_BASE/*.ome.tif*`) match the exclude here? (`$OUTPUT/*.ome.tif*`)

(And if consistency on the the number of "f"s is possible, that would also be good.)

Reply:
In general we probably can't expect this to be universally consistent. They are interchangable.  If you want, I can change the current inputs but we probably want to leave the wildcards.